routes/index.js

```diff
@@ -6,44 +6,48 @@ const router = express.Router();
 const apiSecret = process.env.API_SECRET;
 const apiKey = process.env.API_KEY;
 const opentok = OpenTok(apiKey, apiSecret);
+const ROOM_SIZE = 4;
 
 let sessionId;
 let token;
 
 router.get('/', (req, res) => {
+  //only called after sessionId has been initalized
+  function finish() {
+    token = opentok.generateToken(sessionId);
+    res.render('index.pug', {
+      title: "bobby",
+      apiKey: apiKey,
+      token: token,
+      sessionId: sessionId
+    });
```

Comment:
A cool thing about ES6 is that if the values are defined, you can simply have the keys and it will pick up the values:

```
    res.render('index.pug', {
      title: "bobby",
      apiKey: apiKey,
      token: token,
      sessionId: sessionId
    });
``` 
can now be
```
    res.render('index.pug', {
      title: "bobby",
      apiKey,
      token,
      sessionId
    });
```